### DISCUSSION TITLE
How to remove the embedded link to the dark exported image

### DISCUSSION THREAD

**Discussion by ghost** [excalidraw/excalidraw#6924] [Q&A]
Hi folks, my question is simple, what should I change in the settings to remove the part indicated in red, after embedding a new drawing in my current file. I have toggled several options on/off but I have not succeeded.
![image](https://github.com/excalidraw/excalidraw/assets/107968926/b47fc8b0-5bd1-4d1f-a9e4-8b13ea4346ea)
Thanks in advance.

**Comment by ad1992** [comment]
@ledzeg this looks like a 3rd party when Excalidraw is integrated so it needs to be fixed in that tool instead. Which tool is this ?

**Reply by ghost** [reply]
The tool is [Obsidian](https://obsidian.md/)

**Comment by zsviczian** [comment]
This behavior depends on the last setting in this screenshot. If you auto-export both dark and light versions of the drawing you will get the second part as well.
![image](https://github.com/excalidraw/excalidraw/assets/14358394/ca552dbd-a40b-408e-ac36-909588751859)

This also depends on the "type of file to insert into the document" setting. If you embed Excalidraw like in the screenshot, then the entire section in the Markdown comment will not be displayed.
![image](https://github.com/excalidraw/excalidraw/assets/14358394/ff51fbb1-b354-4f87-b509-b12a49201e2d)

**Reply by ghost** [reply]
Thanks for your anwswer. I tested on a fresh vault but it still shows that "dark exported image". 
I think it goes by default from some Excalidraw version.
